.github/workflows/dev.yml

```diff
@@ -9,5 +9,8 @@ on:
 
 jobs:
   call-test-workflow:
-    #https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
-    uses: ./.github/workflows/run-tests.yml
+    uses: GNS-Science/nshm-github-actions/.github/workflows/python-run-tests.yml@main
+    with:
+      operating-systems: "['ubuntu-latest']"
```

Comment:
same question as toshi-api about only testing on ubuntu and py10

Reply:
Same answer, we control the environment it's run in, and that's what was tested in the deleted `run-tests.yml`